fpsim/version.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.24.1'
-__versiondate__ = '2023-3-17'
+__version__ = '0.24.2'
+__versiondate__ = '2023-5-2'
```

Comment:
Let's make this version 0.25.0 - definitely qualifies as a medium-large change

Reply:
Version updated to 0.25.0